stylesheets/main.css

```diff
@@ -0,0 +1,125 @@
+.grey-text {
+    color: #818181;
+}
+
+.white-text {
+    color: #fff;
+}
+
+.not-quite-black-text {
+    color: #333;
+}
+
+a {
+    text-decoration: none;
+    font-size: 1.2em;
+    color: #3cb9cb;
+    /* padding: .2em; */
+    /* margin: .2em;
+    border: 2px solid #3cb9cb; */
+}
+
+a:hover {
+    text-decoration-line: underline;
+}
+
+/* a:hover {
+    text-decoration-line: underline;
+    text-decoration-color: #818181;
+    border: 2px solid #818181;
+    border-radius: 10px;
+} */
+
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+header {
+    /* grid-template: min-content / 1fr min-content; */
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    align-items: center;
+    background-color: #3cb9cb;
+    padding: 20px 60px;
+    border-bottom: 3px solid #818181;
+}
+span.logo {
+    font-family: 'Crimson Text', serif;
+    font-size: 5em;
+    border-radius: 20px;
+   
+    border: 3px solid #818181;
+    background-color: white;
+    padding: 0 10px;
+}
+
+ul{
+    padding: 0 20px 0 20px;
+}
+
+nav ul li {
+    list-style-type: none;
+    text-align: center;
+    padding: .2em;
+    margin: .2em;
+}
+
+nav ul li:hover {
+     list-style-type: disc;
+}
+
+.flex-box-column {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 2em;
+    min-height: 300px;
+    text-align: left;
+}
+
+.flex-box-row {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 2em;
+    min-height: 300px;
+    text-align: left;
+}
+
+iframe {
+    border: 2px solid #333;
+}
+
+main > h3 {
+    font-size: 1.6em;
+    border-top: #818181 3px solid;
+    padding-top: .4em;
+}
+
+main h3:first-child {
+    border-top: none;
+    padding: 0;
+}
+
+.home-grid {
+    display: grid;
+    grid-template: 1fr 3fr / 1fr;
+}
+
+
+
+footer {
+    background-color: #3cb9cb;
+    text-align: center;
+    padding: .3em;
+    border-top: 3px solid #818181;
+
+}
+/* FIGURE OUT HOW TO MAKE THE <a> TAG NOT HAVE A BULLET UPON HOVER 
+THIS DOESN'T DO ANYTHING */
+nav ul .grey-text li:hover {
```

Comment:
You likely would want to delete the `.grey-text` selector here. You are selecting on all `li` elements when they are on `hover`... so I actually think `li:hover` might work for your purposes!

`nav ul li:hover` is saying "this rule applies to... all `li` tags that are nested under `nav ul`, and when these `li` tags are on hover"